src/Nuget/Sarif.Multitool.nuspec

```diff
@@ -0,0 +1,33 @@
+﻿<?xml version="1.0"?>
+<package>
+  <metadata>
+    <id>Sarif.Multitool</id>
+    <version>$version$</version>
+    <title>Microsoft SARIF Multitool (includes SARIF SDK)</title>
+    <authors>Microsoft Corporation</authors>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>SARIF-based classes and utilities for building static analysis drivers. Includes the SARIF SDK.</description>
+    <releaseNotes>Version $version$ of the .NET SARIF Multitool (for SARIF v1.0.0)</releaseNotes>
+    <copyright>Copyright Microsoft 2015</copyright>
+    <licenseUrl>https://github.com/Microsoft/sarif-sdk/blob/master/LICENSE</licenseUrl>
+    <projectUrl>https://github.com/microsoft/sarif-sdk</projectUrl>
+    <tags>command line driver utilities static analysis sarif framework utils driver.utilities</tags>
```

Comment:
These tags are also copy/pasted from the SDK. Not all of them are appropriate.

Reply:
How about `SARIF command line static analysis`

Reply:
Done